packages/camera/camera_android_camerax/README.md

```diff
@@ -1,3 +1,64 @@
-# camera_android_camerax
+# camera\_android\_camerax
 
-An implementation of the camera plugin on Android using CameraX.
+An Android implementation of [`camera`][1] that uses the [CameraX library][2].
+
+*Note*: This package is under development.
```

Comment:
What should someone reading this take away from "This is under development"? 

Should they be excited for a rapid set of updated features? 
Should they expect more urgent handling of reported bugs? 
Should they avoid usage because the api is still in flux? 
Does this mean it is not production ready? 

Consider using phrasing that tells a developer what they should take away instead of saying it is under development.